code/game/gamemodes/changeling/powers/biodegrade.dm

```diff
@@ -54,17 +54,20 @@
 
 /obj/effect/proc_holder/changeling/biodegrade/proc/dissolve_handcuffs(mob/living/carbon/human/user, obj/O)
 	if(O && user.handcuffed == O)
-		visible_message("<span class='warning'>[O] dissolves into a puddle of sizzling goop.</span>")
+		user.visible_message("<span class='warning'>[O] dissolve[O.gender==PLURAL?"":"s"] into a puddle of sizzling goop.</span>")
```

Comment:
I could have sworn there was a byond feature for adding s onto the end of words if the item is plural

Reply:
Yes, it's \s. 

This is doing the opposite however, removing the s in dissolves if the object is plural as you are turning the plural noun into a singular (a puddle), as opposed to keeping it a plural (multiple puddles, in which case you would drop the a and retain the s in dissolves.)

This is functionality is performed for /mob/ using the p_s() proc, however as I have multiple PRs doing the same thing for the most part, I'll follow up afterward with another PR and add the necessary procs for /obj/ since there are a few additional cases of similar pronoun code being used in other areas that would need to be cleaned up as well.

Reply:
Ah, I see now

Reply:
I chose to identify as plural